app/assets/javascripts/dictionary.js.coffee

```diff
@@ -18,3 +18,28 @@ doTestField = ->
 
 $ ->
   $('#field_name_test').on('input', doTestField)
+
+
+# Sets up the sidenav on the dictionary.
+# Must be called on the page, or else the script misbehaves
+window.dictionarySetupSidenav = ->
```

Comment:
Tiny cleanup idea:

- Declare `dictionarySetupSidenav` without attaching it to the window object.
- Call dictionarySetupSidenav from the `$ ->` function.  That function gets called automatically when the page finishes loading.
- Then you can remove the `<script>` block from `_index_upper_content.html.erb`.